cmd/plasmacli/subcmd/eth/status.go

```diff
@@ -0,0 +1,45 @@
+package eth
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/cmd/plasmacli/config"
+	"github.com/FourthState/plasma-mvp-sidechain/eth"
+	"github.com/spf13/cobra"
+)
+// StatusCmd returns the current state of the eth connection (syncing, crashed etc)
+func StatusCmd() *cobra.Command {
+	return statuscmd
+}
+
+var statuscmd = &cobra.Command {
+	Use:   "status",
+	Short: "check state of eth connection",
+	Long: `returns current state of eth connection (syncing, crashed, etc)`,
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		// Get Conf
+		conf, err := config.ParseConfigFromViper()
+		if err != nil {
+			return fmt.Errorf("error retrieving config: { %s }", err)
+		}
+
+		// Initialize Eth Connection
+		client, err := eth.InitEthConn(conf.EthNodeURL)
+		if err != nil {
+			return fmt.Errorf("error initializing connection with client: { %s }", err)
```

Comment:
I think the `ParseConfigFromViper()` and `InitEthConn` can be replaced with a call to [config.GetContractConn()](https://github.com/FourthState/plasma-mvp-sidechain/blob/536e4ac2156bfa6add7fc76da3ce123820437fc8/cmd/plasmacli/config/connection.go#L14)

It does the same thing (DRY) so might as well use the library and the library has verbose error messages.

Reply:
Ah good catch, that's right

Reply:
Ah wait, `config.GetContractConn` provides the `eth.Plasma` struct, in which the ethereum client is a private field. 

Is it worth trying to improve this?

Reply:
@colin-axner will leave it to you to make the call!

Reply:
Let's leave it how you have it for now for the reason I listed. Unless you have other opinions @colin-axner.

Reply:
I'm fine with an issue being opened to do these changes later, code can be improved but functionality wouldn't change so not a blocking request